src/frontends/tensorflow/tests/requirements.txt

```diff
@@ -1,2 +1,2 @@
 numpy>=1.16.6,<=1.23.4
-tensorflow>=1.15.5,<=2.11.0
+tensorflow>=1.15.5,<=2.13.0rc0,!=2.13.0rc0
```

Comment:
We have to use release candidates, because TensorFlow 2.11 has protobuf requirements which are incompatible with ONNX, they are older. TensorFlow 2.12, which release candidates are already on PyPI, relaxes the protobuf requirements and allows us to proceed.

The requirement is written in a way which will install full TensorFlow 2.12 release once it's on PyPI. The only other option to go forward with this PR (and by extension Python 3.11) is to wait for the official release. I insist on using release candidates.

Reply:
@p-wysocki +1 on this. There is no other way to overcome "rc" tag in plain pip. This is the only working solution that will fit our project.

Reply:
It looks that you did these changes to get tests passed and due to our limitations in the validation. Because our validation requires to have installed dependencies for all FWs for conversion of different FW models. So they become incompatible and cannot exist together.

May be we should solve this problem on the validation part and not to change production `requirements` files with RC dependencies. Also, now our customer can potentially install RC on his machine. That is not good!

Reply:
to resolve a discussion - TF 2.12 which satisfies our requirements has been released and this notation is not needed anymore and will be replaced with released-based one.